solidity/.eslintrc

```diff
@@ -0,0 +1,15 @@
+{
+    "extends": ["eslint-config-keep", "prettier"],
```

Comment:
`"prettier"` here could probably be removed as it's already included in `"eslint-config-keep"` config.